src/Behat/Testwork/Specification/SpecificationArrayIterator.php

```diff
@@ -19,6 +19,7 @@
  * Return instance of this class from locator if specifications cannot be searched lazily.
  *
  * @author Christophe Coevoet <stof@notk.org>
+ * @extends ArrayIterator<int, mixed>
```

Comment:
`ArrayIterator` is defined as a template, so we need to specify the concrete type when extending it